apps/condo/domains/notification/schema/MessageBatch.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Generated by `createschema notification.MessageBatch 'messageType:Text; title:Text; message:Text; deepLink?:Text; targets:Json; status:Select:created,processing,failed,done; processingMeta?:Json;'`
+ */
+
+const { isArray, isEmpty } = require('lodash')
+
+const { Text, Select } = require('@keystonejs/fields')
+
+const { Json } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+
+const { hasValidJsonStructure } = require('@condo/domains/common/utils/validation.utils')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+
+const { JSON_EXPECT_ARRAY_ERROR, OPERATION_FORBIDDEN } = require('@condo/domains/common/constants/errors')
+
+const access = require('@condo/domains/notification/access/MessageBatch')
+const {
+    CUSTOM_CONTENT_MESSAGE_TYPE,
+    MESSAGE_BATCH_STATUSES,
+    MESSAGE_BATCH_CREATED_STATUS,
+    MESSAGE_BATCH_TYPE_OPTIONS,
+} = require('@condo/domains/notification/constants/constants')
+
+const { sendMessageBatch } = require('../tasks/sendMessageBatch')
+
+const operationForbiddenValidator = ({ fieldPath, addFieldValidationError, operation }) => {
+    if (operation === 'update') addFieldValidationError(`${OPERATION_FORBIDDEN}] Updating ${fieldPath} is forbidden.`)
+}
+
+const MessageBatch = new GQLListSchema('MessageBatch', {
```

Comment:
A name `MessageBatch` does not reflects it's primary side effect.

```suggestion
const MessageBatchSendingTask = new GQLListSchema('MessageBatch', {
```

Reply:
Actually it does. This is batch, it is handled by async task.

Reply:
MessageBatch is a noun, it's primary side-effect is an action — verb.
There is no verb in name.